src/org/zaproxy/zap/extension/ascanrulesBeta/ZapAddOn.xml

```diff
@@ -6,6 +6,10 @@
 	<url/>
 	<changes>
 	<![CDATA[
+	Minor code changes.<br>
+	Change scanners to honour the technologies enabled (Issue 1618).<br>
+	Added Buffer Overflow scanner to beta (Issue 1605).<br>
```

Comment:
OK. Note that you can change the upstream of a branch any time (with ```git branch -u``` or even in Eclipse).